build.gradle

```diff
@@ -111,6 +111,38 @@ static Map<String, String> getEnvironmentVariables(String... envNames) {
     return envNames.collectEntries {envName -> [(envName): System.getenv(envName)] }
 }
 
+String getMavenRepositoryURL() {
+    def url = "${environment.OBLIVIUM_MAVEN_URL}/${environment.OBLIVIUM_MAVEN_REPOSITORY}"
+    if (conclave_version.endsWith("-SNAPSHOT")) {
```

Comment:
Use the version string for conclave graal, not of Conclave. Yes, for this check they are equivalent, but it's clearer to use the version of the project that is going to be published.